lib/jobs/swift_review_job.rb

```
@@ -16,27 +17,39 @@ def self.perform(attributes)
     # content
     # config
 
-    violations = violations_for_content(attributes["content"])
+    file = file_for(attributes: attributes)
+    violations = violations_for(file: file)
 
+    completed_file_review(
+      file: file,
+      attributes: attributes,
+      violations: violations,
+    )
+  rescue => error
```

I think everything currently. Because this all runs inside of a job, the errors are all swallowed so you can't see what's going on. I'd like to add on an error tracker at some point cause even with this you likely won't see it.